editor/src/clj/editor/scene.clj

```diff
@@ -348,11 +354,12 @@
                                    :selected appear-selected?
                                    :user-data (:user-data renderable)
                                    :batch-key (:batch-key renderable)
-                                   :aabb (geom/aabb-transform ^AABB (:aabb scene geom/null-aabb) world-transform)
+                                   :aabb (geom/aabb-transform aabb world-transform)
                                    :render-key (render-key view-proj world-transform (:index renderable) (:topmost? renderable))
                                    :pass-overrides {pass/outline {:render-key (outline-render-key view-proj world-transform (:index renderable) (:topmost? renderable) appear-selected?)}}))
-        visible? (and (not (contains? hidden-node-outline-key-paths node-outline-key-path))
-                      (not-any? (partial contains? hidden-renderable-tags) (:tags flat-renderable)))
+        flat-renderable (if visible?
+                          flat-renderable
+                          (dissoc flat-renderable :updatable))
```

Comment:
Remove any `:updatable` from hidden objects. This will stop continuously updating stuff like particle effects from rendering.